rpc/src/rpc.rs

```diff
@@ -284,6 +284,13 @@ impl JsonRpcRequestProcessor {
         Ok(bank)
     }
 
+    fn check_if_transaction_history_enabled(&self) -> Result<()> {
```

Comment:
nit: Is it conventional in Rust to call these ‘checkers’ or is it appropriate to call them assertions, even though they don't `panic!` but rather return a `Result?
```suggestion
    fn assert_transaction_history_enabled(&self) -> Result<()> {
```

Reply:
To me, anything with `assert` in the name may panic. I landed on this name for sake of consistency with the existing `check_is_at_least_confirmed()`. Not positive these are the most "idiomatic", but at least they're consistent 😄